usersync/chooser.go

```diff
@@ -177,9 +184,17 @@ func (c standardChooser) evaluate(bidder string, syncersSeen map[string]struct{}
 		return nil, BidderEvaluation{Status: StatusBlockedByGDPR, Bidder: bidder, SyncerKey: syncer.Key()}
 	}
 
+	if c.bidderInfo[bidder].Syncer != nil && c.bidderInfo[bidder].Syncer.SkipWhen != nil && c.bidderInfo[bidder].Syncer.SkipWhen.GDPR {
```

Comment:
Shouldn't there be a check here to see if the GDPR applies? Otherwise we will always skip every request when they ask to skip when the GDPR applies. May want to put this in front of the GDPR check above so we don't bother checking if they pass the GDPR if they want to skip any GDPR requests.